packages/opentelemetry-sdk-trace-web/src/utils.ts

```diff
@@ -125,6 +129,155 @@ export function addSpanNetworkEvents(
   }
 }
 
+function _getBodyNonDestructively(body: ReadableStream) {
```

Comment:
> My guess is that the reason this requires changes to @opentelemetry/sdk-trace-web is that we're trying to:
>
> * reduce added bundle size for when both instrumentations are added
> * reduce duplicated code and tests for the same functionality.

Yeah that's exactly correct. I can duplicate the logic across both packages if you'd prefer that.

I think it could make sense to unify the two packages. Both packages deal with network instrumentation and, given the state of the web, I would assume that most projects have a mix of fetch and XHR calls in their code. I would be surprised if there were a lot of people intentionally using only one of the 2 instrumentations. 

But unifying them is a higher level decision that I'll defer to you and the other maintainers :)

Reply:
Let's duplicate for now. Timeline for removing it from this package would likely be > 1 year. Duplicating is not the best choice, but it's better than being stuck as part of the public API - it gives us an "out" earlier than that.

Combining the packages is something that needs to be discussed with the SIG first - I'll put it on the agenda for next week's SIG meeting. If accepted, we can follow through with it after this PR has merged.